pkgs/tools/cd-dvd/cdrtools/default.nix

```diff
@@ -12,6 +12,11 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ m4 ];
   buildInputs = if stdenv.hostPlatform.isDarwin then [ Carbon IOKit ] else [ acl libcap ];
 
+  env.CFLAGS = toString [
+    "-Wno-error=implicit-int"
+    "-Wno-error=implicit-function-declaration"
+  ];
```

Comment:
Non‐blocking nit: `lib.concatStringsSep " "` has less surprising behaviour in general (but they are equivalent in this case).